paper-radio-group.html

```diff
@@ -27,7 +27,7 @@
       <paper-radio-button name="large" label="Large"></paper-radio-button>
     </paper-radio-group>
 
-See <a href="paper-radio-button.html">paper-radio-button</a> for more
+See <a href="paper-radio-button">paper-radio-button</a> for more
```

Comment:
Actually, would you mind changing this to not be a link, and just say "See `Polymer.PaperRadioButton` for ..."?